test/test_train_mnist.py

```diff
@@ -163,6 +166,8 @@ def test_loop_fn(model, loader, device, context):
     if FLAGS.metrics_debug:
       print(met.metrics_report())
 
+  if writer:
```

Comment:
This seems pretty ugly. Are we sure we need to do this?

Reply:
probably not strictly necessary for imagenet since the data auto-flushes every 120sec, but for mnist and cifar the summary will be missing data unless we do this:
https://github.com/pytorch/pytorch/issues/24234

Reply:
Switched to a util method